src/track/serato/cueinfoimporter.h

```diff
@@ -0,0 +1,15 @@
+#pragma once
+#include "track/cueinfoimporter.h"
+
+namespace mixxx {
+
+class SeratoCueInfoImporter : public CueInfoImporter {
+  public:
+    using CueInfoImporter::CueInfoImporter;
```

Comment:
~SeratoCueInfoImporter() override = default;

Reply:
Shoudn't it just use the default destructor from the parent class?

Reply:
Best practice, just to ensure that the destructor of the base class is virtual. Safe C++ is verbose.